bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -3194,10 +3196,29 @@ public static SleuthkitCase openCase(String dbPath, ContentStreamProvider provid
 	 * @throws org.sleuthkit.datamodel.TskCoreException
 	 */
 	@Beta
-	public static SleuthkitCase openCase(String dbPath, ContentStreamProvider provider, String lockingApplicationName) throws TskCoreException {
+	public static SleuthkitCase openCase(String dbPath, ContentStreamProvider contentProvider, String lockingApplicationName) throws TskCoreException {
+		return openCase(dbPath, contentProvider, null, false);
+	}
+	
+	/**
+	 * Open an existing case database.
+	 *
+	 * @param dbPath Path to SQLite case database.
+	 * @param contentProvider Custom provider for file content bytes (can be null).
+	 * @param lockingApplicationName The name of the application locking the
+	 *                               case database (null value prevents
+	 *                               locking; 500 character maximum).
+	 * 
+	 * @param useWAL				 Flag to set journal_mode=WAL
```

Comment:
Perhaps add a comment here (or a few lines earlier) about not using WAL for network drives (so we don't forget and have this discussion again in a year)

Reply:
done

Reply:
Perfect